installer/pyinstaller/build-linux.sh

```diff
@@ -47,15 +57,34 @@ echo "Installing PyInstaller"
 
 echo "Building Binary"
 cd src
+if [ "$is_nightly" = "true" ]; then
+    echo "Updating samcli.spec with nightly build"
+    sed -i.bak "s/'sam'/'sam-nightly'/g" installer/pyinstaller/samcli.spec
+    rm installer/pyinstaller/samcli.spec.bak
```

Comment:
I'd put this in a separate function to make it clear what it does and prevent human error on the names. e.g.

```shell
sed_inplace() {
    sed -i.bak "$1" "$2"
    rm "$2.bak"
}
```

Reply:
Thanks, I agree that making the script into function is a good idea. Since this is a common usage to remove backup. I will keep this to save testing effort. Will use this as a function if we need more in-place changes in the future.